notificator/notificator_queue.py

```diff
@@ -80,14 +80,21 @@ async def _is_already_notified(self, rh_account_id, cve_id, event_type):
             self.notified_accounts[(rh_account_id, cve_id)] = new_events
         return event_type in new_events
 
-    @staticmethod
-    def _create_notif_events(cve):
+    def _create_notif_events(self, cve_id):
         """Create list of events for notifications"""
+        cve = self.conditions.get_cve(cve_id)
+        if not cve:
+            return []
         return [
             {
                 "metadata": {},
                 "payload": {
```

Comment:
@yungbender would it be possible to add account_id/org_id as well? (Likely to context, but it can stay in payload with out an issue). Thank you.